sw/device/tests/flash_ctrl_clock_freqs_test.c

```diff
@@ -140,10 +141,12 @@ bool test_main(void) {
       mmio_region_from_addr(TOP_EARLGREY_FLASH_CTRL_CORE_BASE_ADDR)));
 
   for (int i = 0; i < kNumLoops; ++i) {
-    do_info_partition_test(kFlashInfoPageIdCreatorSecret);
-    do_info_partition_test(kFlashInfoPageIdOwnerSecret);
-    do_info_partition_test(kFlashInfoPageIdIsoPart);
-    do_data_partition_test(kFlashDataBank0);
+    if (kBootStage != kBootStageOwner) {
```

Comment:
should we also check the device type here? I don't think we want to overwrite the creator or owner secret pages on silicon as overwriting these on a theoretically provisioned device would alter the key ladder and invalidate certs. though I suppose we expect such a device type would have a rom_ext in place?

Reply:
> should we also check the device type here? I don't think we want to overwrite the creator or owner secret pages on silicon as overwriting these on a theoretically provisioned device would alter the key ladder and invalidate certs. though I suppose we expect such a device type would have a rom_ext in place?

Replaced all (kBootStage, kBootStageOwner) with (kDeviceType , kDeviceSilicon).